src/platform/mt8195/include/platform/drivers/interrupt.h

```diff
@@ -0,0 +1,183 @@
+/* SPDX-License-Identifier: BSD-3-Clause
+ *
+ * Copyright(c) 2021 Mediatek
+ *
+ * Author: Hailong Fan <hailong.fan@mediatek.com>
+ */
+
+#ifdef __SOF_DRIVERS_INTERRUPT_H__
+
+#ifndef __PLATFORM_DRIVERS_INTERRUPT_H__
+#define __PLATFORM_DRIVERS_INTERRUPT_H__
+
+#include <stdint.h>
+#include <platform/drivers/mt_reg_base.h>
+
+#define io_reg_set(addr, data)		((*(volatile unsigned int *)(addr)) |= (unsigned int)(data))
+#define io_reg_clr(addr, data)		((*(volatile unsigned int *)(addr)) &= ~((unsigned int)(data)))
```

Comment:
Do some operations really require these, instead of using the standard io_reg_read/write/update_bits from <sof/lib/io.h>?